src/sentry/options/defaults.py

```diff
@@ -13,13 +13,13 @@
 from sentry.utils.types import Dict, String
 
 # Cache
-register('cache.backend', flags=FLAG_NOSTORE)
-register('cache.options', type=Dict, flags=FLAG_NOSTORE)
+# register('cache.backend', flags=FLAG_NOSTORE)
```

Comment:
I'm explicitly commenting these out since they weren't being used, and would add to confusion here.